sdk/core/System.ClientModel/src/Message/HttpClientResponseHeaders.cs

```diff
@@ -21,10 +22,18 @@ public HttpClientResponseHeaders(HttpResponseMessage response, HttpContent respo
     }
 
     public override bool TryGetValue(string name, out string? value)
-        => TryGetHeader(_httpResponse.Headers, _httpResponseContent, name, out value);
+    {
+        Argument.AssertNotNull(name, nameof(name));
```

Comment:
Do we want to allow `string.Empty` for these?

Reply:
Addressed.